src/app/routes/utils/constructPageFetchUrl/index.ts

```diff
@@ -215,7 +231,7 @@ const constructPageFetchUrl = ({
         const host = `http://${process.env.HOSTNAME || 'localhost'}`;
         const port = process.env.PORT ? `:${process.env.PORT}` : '';
         fetchUrl = Url(
-          `${host}${port}/api/local/${service}/live/${id}${variantPath}`,
+          `${host}${port}/api/local/${service}/live/${pathname}${variantPath}`,
```

Comment:
Could we add a comment here to explain?

Something along the lines of: pathname is the ID of the live page, without `/service/live/`, and supports both Tipo & CPS IDs?